libr/core/cconfig.c

```diff
@@ -3018,6 +3018,8 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF ("http.upget", "false", "/up/ answers GET requests, in addition to POST");
 	SETPREF ("http.upload", "false", "Enable file uploads to /up/<filename>");
 	SETPREF ("http.uri", "", "Address of HTTP proxy");
+	SETPREF ("http.auth", "false", "Enable/Disable HTTP Authentification");
+	SETPREF ("http.authtok", "r2admin:r2admin", "HTTP Authentification user:password token");
```

Comment:
what about http.auth.file or cmd.httpauth?

Reply:
Yes in multi user context having a file makes more sense. If you insist I ll make the necessary change for multi user usage (see original PR comment).